### ISSUE TITLE
RuntimeException: X is not a link to Y (due to directory separator issue)

### ISSUE THREAD

**Issue by fritzmg** [contao-community-alliance/composer-plugin#53]
I tried to uninstall a package (which is a Contao 3 extension made compatible with Contao 4). When the composer-plugin tries to remove the symlink from the `/system/modules` directory, the following error happens (under Windows):
```
[RuntimeException]
"C:/xampp/htdocs/c41dev/system/modules/inherit_article" is not a link to "C:/xampp/htdocs/c41dev/vendor/fritzmg/contao-inherit-article/system/modules/inherit_article"
```
However, that error message is not correct, here is the `dir /a` output from the `/system/modules` directory:
```
C:\xampp\htdocs\c41dev\system\modules>dir /a
 Volume in drive C has no label.
 Volume Serial Number is F898-24D6

 Directory of C:\xampp\htdocs\c41dev\system\modules

15.02.2016  13:04    <DIR>          .
15.02.2016  13:04    <DIR>          ..
15.02.2016  12:51                57 .gitignore
15.02.2016  13:04    <SYMLINKD>     inherit_article [C:/xampp/htdocs/c41dev/vendor/fritzmg/contao-inherit-article/system/modules/inherit_article]
               1 File(s)             57 bytes
               3 Dir(s)  93.502.509.056 bytes free
```
As you can see contrary to what the error message says `C:/xampp/htdocs/c41dev/system/modules/inherit_article` is indeed a link to `C:/xampp/htdocs/c41dev/vendor/fritzmg/contao-inherit-article/system/modules/inherit_article`.

Doing
```php
var_dump($source);
var_dump($target);
var_dump(readlink($target));
```
in `AbstractModuleInstlaler::canRemoveSymlink` yields the following:
```
string(91) "C:/xampp/htdocs/c41dev/vendor/fritzmg/contao-inherit-article/system/modules/inherit_article"
string(53) "C:/xampp/htdocs/c41dev/system/modules/inherit_article"
string(91) "C:\xampp\htdocs\c41dev\vendor\fritzmg\contao-inherit-article\system\modules\inherit_article"
```
So even though the symlink is correct, the function `canRemoveSymlink` fails because it does not normalize the directory separators.

**Comment by aschempp** [comment]
Please try the fix in 6637140639a300bf4d68f5b6b64b6778fc65a8d0

**Comment by fritzmg** [comment]
Yep, seems to work just fine :)
